api/src/services/script-runner.service.ts

```diff
@@ -651,6 +651,34 @@ export class ScriptRunnerService {
     return { success: true };
   }
 
+  /**
+   *
+   * @param req incoming request object
+   * @returns successDTO
+   * @description copies the updateAt timestamp to contentUpdatedAt for Detroit listings table
+   */
+  async copyDetroitUpdatedAtToContentUpdatedAt(
+    req: ExpressRequest,
+  ): Promise<SuccessDTO> {
+    const requestingUser = mapTo(User, req['user']);
+    await this.markScriptAsRunStart(
+      'copy Detroit updatedAt timestamp to contentUpdatedAt',
+      requestingUser,
+    );
+
+    await this.prisma.$queryRawUnsafe(`
+      UPDATE listings
+      SET content_updated_at = updated_at
+      WHERE content_updated_at IS NULL;
+    `);
```

Comment:
Being that nothing outside of a postgres script is needed I would prefer if we have this as a migration file. Ideally we wouldn't add any script runners to this repo so that the backend code doesn't have any merge conflicts. Migration files are full new files so we would never run into merging conflicts